services/_example_service.md

```diff
@@ -0,0 +1,87 @@
+| in   | cmd.domain_a.get_report | null       |              |              | Requests a report of something from domain A.                                         |
+| in   | cmd.domain_a.set        | object     |              |              | Sets something in domain A, see definition of [`object_a`](#definitions) object.      |
+| out  | evt.domain_a.report     | object     |              |              | Reports something from domain A, see definition of [`object_a`](#definitions) object. |
+| -    |                         |            |              |              |                                                                                       |
+| in   | cmd.domain_b.get_report | string     |              |              | Requests a report of something from domain B.                                         |
+| out  | evt.domain_b.report     | float      | `property_a` | `property_a` | Reports something from domain B.                                                      |
+
+## Interface properties
+
+| Name         | Example   | Description                                                                                                     |
+|--------------|-----------|-----------------------------------------------------------------------------------------------------------------|
+| `property_a` | `"val_1"` | This is an example value, one of possible values defined in [`sup_vals`](#service-properties) service property. |
+
+> Note that all interface properties included in the FIMP message must be strings as per the message schema.
+
+## Service properties
+
+| Name       | Type      | Example              | Description                                                                                      |
+|------------|-----------|----------------------|--------------------------------------------------------------------------------------------------|
+| `sup_vals` | str_array | `["val_1", "val_2"]` | Description of the service property, may contain one or more values of [`enum_a`](#definitions). |
+
+> Note that each service property included in the device inclusion report can have a different type.
+
+## Definitions
+
+* `attribute_a` is an object with the following structure:
+
+| Field   | Type   | Example     | Description                                                       |
+|---------|--------|-------------|-------------------------------------------------------------------|
+| field_a | float  | `123.45`    | Description of the field.                                         |
+| field_b | string | `"value_2"` | Description of the field, one of [`enum_a`](#definitions) values. |
+
+* `enum_a` is one of the following values: `val_1`, `val_2`, `val_3`.
+
+## Examples
+
+* Example of a report of something from domain A:
+
+```json
+{
+  "serv": "example",
+  "type": "evt.domain_a.report",
+  "val_t": "object",
+  "val": {
+    "field_a": 123.45,
+    "field_b": "value_2"
+  }
+}
```

Comment:
I understand that we don't want our examples bloated, but should they maybe include all required fields, such as `ver`, `src`, `ctime` and `uid`? Just to avoid encouraging sending any FIMP messages without them, I mean

Reply:
hmm, I think it's safer to include them then, better than having an example that's actually not a valid message, I would say

Reply:
Will do it in next PR.